platform/solr/solr-factory/src/main/java/org/codice/solr/factory/impl/SolrHttpRequestRetryHandler.java

```diff
@@ -13,46 +13,85 @@
  */
 package org.codice.solr.factory.impl;
 
+import com.google.common.annotations.VisibleForTesting;
 import java.io.IOException;
 import java.io.InterruptedIOException;
 import java.net.UnknownHostException;
 import javax.net.ssl.SSLException;
+import org.apache.commons.lang.Validate;
 import org.apache.commons.lang.time.DurationFormatUtils;
 import org.apache.http.client.HttpRequestRetryHandler;
 import org.apache.http.protocol.HttpContext;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class SolrHttpRequestRetryHandler implements HttpRequestRetryHandler {
```

Comment:
❓package-private instead of `public`?
✏️JavaDoc comment would be nice too :-)

Reply:
It took you 3 years to complain about the missing javadoc? I thought that a good name didn't need a comment. 😄